docs/sql-reference/sql-functions/date-time-functions/date_diff.md

```diff
@@ -0,0 +1,49 @@
+# date_diff
+
+## Description
+
+Returns the ISO standard weekday number for a given date.
+
+The date parameter must be of the DATE or DATETIME or VARCHAR type.
+
+## Syntax
+
+```Haskell
+BIGINT DATE_DIFF(DATETIME L, DATETIME R, VARCHAR TYPE)
```

Comment:
It seems the third paramter can only be certain values like 'hour', 'minute'... You need check the third parameter in the analyze phase.